protokube/cmd/protokube/main.go

```diff
@@ -98,6 +98,9 @@ func run() error {
 	var gossipSecret string
 	flags.StringVar(&gossipSecret, "gossip-secret", gossipSecret, "Secret to use to secure gossip")
 
+	var etcdImageRepository string
+	flags.StringVar(&etcdImageRepository, "etcd-image-repository", etcdImageRepository, "Etd Container url")
```

Comment:
Type: etd

Maybe `etcd-image-source`, as that could cover a filesystem also, but I'll see how you're using it...